crates/nu-command/src/filters/default.rs

```diff
@@ -146,6 +147,20 @@ fn default(
             && matches!(input, PipelineData::Value(ref value, _) if value.is_empty()))
     {
         Ok(value.into_pipeline_data())
+    } else if default_when_empty && matches!(input, PipelineData::ListStream(..)) {
+        let PipelineData::ListStream(ls, metadata) = input else {
+            unreachable!()
+        };
+        let span = ls.span();
+        let mut stream = ls.into_inner().peekable();
+        if stream.peek().is_none() {
+            return Ok(value.into_pipeline_data());
+        }
+
+        // stream's internal state already preserves the original signals config, so if this
+        // Signals::empty list stream gets interrupted it will be catched by the underlying iterator
```

Comment:
This typo fix and a `cargo fmt --all` and we should be good to go :)

```suggestion
        // Signals::empty list stream gets interrupted it will be caught by the underlying iterator
```